src/minecraft/handlers/ChatHandler.js

```diff
@@ -209,19 +209,28 @@ class StateHandler extends EventHandler {
           let user = message.split(' ')[1]
           user = user.replace('-----------------------------------------------------', '')
           user = user.replace('\n', '')
-          if (config.discord.autoAccept===true){
-            this.bot.chat(`/g accept ${user}`)
-          }
+          if (config.discord.autoAccept===true){  
+            const blacklistedPlayers = config.discord.blacklistedPlayers
+            if (blacklistedPlayers.includes(user)) {
+              this.bot.chat(`/msg ${user} You are blacklisted from joining SkyKings guilds!`);
```

Comment:
Probably just ignore the join request instead of messaging them to prevent possible spam